src/index.js

```diff
@@ -46,7 +46,42 @@ registerFormatType( type, {
 		};
 		// Pin the Popover to the caret position.
 		const anchorRect = () => {
-			return getRectangleFromRange( anchorRange );
+			/**
```

Comment:
```suggestion
			/*
```

Nitpick: Only docblocks use `**`, multi-line comments use a single asterisks.

Reply:
@peterwilsoncc I made this change. Out of curiosity can you also explain on the rationale behind this?

Reply:
> can you also explain on the rationale behind this?

The short version is that it [meets the WP coding standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments) but I can take an educated guess as to the reason behind it.

The [phpdoc spec](https://docs.phpdoc.org/guide/getting-started/what-is-a-docblock.html#what-does-a-docblock-look-like) defines a docblock as starting with a double asterisks. The JSDoc spec uses the same convention.

My guess is that it helps with parsing the code files in to something more useful, such as the developer.wordpress.org pages. If a comment begins with `/**` then the parser knows it needs to check whether the comment is a valid docblock.

If a comment only begins with `/*` then the parser knows it can be skipped and considered an inline comment.